package.json

```diff
@@ -13,7 +13,7 @@
     "linter"
   ],
   "scripts": {
-    "test": "coffeelint ./lib/main.coffee"
+    "lint": "./node_modules/.bin/coffeelint ./lib/main.coffee"
```

Comment:
npm exports the `node_modules/.bin` folder to the PATH when running scripts ([see here](https://docs.npmjs.com/misc/scripts#path)), so you should be able to just leave that as `coffeelint`.

It should work either way though :wink:.